oauthproxy.go

```diff
@@ -1098,14 +1110,17 @@ func (p *OAuthProxy) CheckBasicAuth(req *http.Request) (*sessionsapi.SessionStat
 
 // isAjax checks if a request is an ajax request
 func isAjax(req *http.Request) bool {
-	acceptValues := req.Header.Values("Accept")
-	const ajaxReq = applicationJSON
-	for _, v := range acceptValues {
-		if v == ajaxReq {
-			return true
-		}
-	}
-	return false
+        acceptValues, ok := req.Header["accept"]
```

Comment:
- use the same syntax of values as earlier. Only you need check for "accept" and "Accept" which you are doing here

Reply:
try only check fro "accept" getting error req.Header.Values undefined (type http.Header has no field or method Values)